Makefile

```diff
@@ -33,6 +33,8 @@ GOFILES := go list  -f '{{join .Deps "\n"}}' ./cmd/minishift/ | grep $(REPOPATH)
 PACKAGES := go list ./... | grep -v /vendor
 SOURCE_DIRS = cmd pkg test
 
+MINISHIFT_GITHUB_API_TOKEN = $(shell echo $MINISHIFT_GITHUB_API_TOKEN)
```

Comment:
where are we using this variable? or this is for API rate limit issue on travis?